install_hooks.py

```diff
@@ -16,25 +59,60 @@ def get_init_scripts(config):
     elif 'bsd' in sys.platform or 'dragonfly' in sys.platform:
         data_files = data_files + "\nusr/local/etc/rc.d = bin/rc.d/*"
     else:
-        raise "Unsupported platform, sorry"
-        data_files = []
+        raise Exception("Unsupported platform, sorry")
+
     config['files']['data_files'] = data_files
 
+    for line in config['files']['data_files'].split('\n'):
+        line = line.strip().split('=')
+        print("Installable directories/files: %s" % line)
+
+    # Check Alignak recommended user existence
+    if not user_exists('alignak'):
```

Comment:
everywhere we say that we need an alignak user ... but nowhere we test if it exists ... but why not. 

Next time please comment on the issues to say that you disagree for a feature: https://github.com/Alignak-monitoring/alignak/issues/247#issuecomment-246606241